binding.gyp

```diff
@@ -6,15 +6,28 @@
   },
   "targets": [
     {
-      "target_name": "<(module_name)",
+      "target_name": "vscode-sqlite3",
       "cflags!": [ "-fno-exceptions" ],
       "cflags_cc!": [ "-fno-exceptions" ],
       "xcode_settings": { "GCC_ENABLE_CPP_EXCEPTIONS": "YES",
         "CLANG_CXX_LIBRARY": "libc++",
-        "MACOSX_DEPLOYMENT_TARGET": "10.7",
+        # Target depends on
+        # https://chromium.googlesource.com/chromium/src/+/master/build/config/mac/mac_sdk.gni#22
+        "MACOSX_DEPLOYMENT_TARGET": "10.11",
```

Comment:
@deepak1556 is this still reflecting our target in VS Code building?

Reply:
Yup for Electron 19 , the deployment target is still `10.11`